app/src/components/main/analytics/events/common/testItemPages/modalEventsCreators.js

```diff
@@ -170,6 +176,11 @@ export const getEditToInvestigateSelectSpecificSimilarItemEvent = (category) =>
 });
 
 // UNLINK ISSUE MODAL
+const getBasicUnlinkIssueModalEventParameters = (category) => ({
```

Comment:
Does it make sense to hold it in the separate function?
As I see it is used only once for `UNLINK_BTN_UNLINK_ISSUE_MODAL`.